src/js/pensions/containers/ConfirmationPage.jsx

```diff
@@ -54,8 +55,18 @@ class ConfirmationPage extends React.Component {
               <strong>Date received</strong><br/>
               <span>{moment(form.submission.submittedAt).format('MMM D, YYYY')}</span>
             </li>
+            <li>
+              <strong>Confirmation number</strong><br/>
+              <span>{response.confirmationNumber}</span>
+            </li>
+            <li>
+              <strong>Your claim was sent to</strong><br/>
+              <address className="schemaform-address-view">{regionalOffice.map((line, index) =>
+                <p key={index}>{line}</p>)}</address>
```

Comment:
Yeah, I think it's overkill for this (and the index really is the key for each item here), but something we should avoid in general. We've had at least one bug due to using index as a key.